traits/tests/test_observe.py

```diff
@@ -930,3 +931,49 @@ class A(HasTraits):
         self.assertEqual(event.index, 2)
         self.assertEqual(event.removed, [3])
         self.assertEqual(event.added, [4])
+
+
+# Integration tests for async observe decorator -------------------------------
+
+
+class SimpleAsyncExample(HasTraits):
+
+    value = Str()
+
+    events = List()
+
+    @observe('value')
+    async def value_changed_async(self, event):
+        await asyncio.sleep(0)
```

Comment:
What's the purpose of the `sleep` here?

Reply:
My understanding is that this yields execution to the event loop - originally I had a longer sleep here, but less than the sleep in the test.  It may not be needed - but I would like the async function to be actually doing something async-y before it sets the trait.